packages/node_modules/@webex/plugin-meetings/src/media/index.js

```diff
@@ -287,28 +287,48 @@ Media.updateTransceiver = ({meetingId, remoteQualityLevel}, peerConnection, tran
  * @returns {Promise.<MediaStream>}
  */
 Media.getDisplayMedia = (options, config = {}) => {
-  const resolutionConf = config.resolution || {};
-  const {screenResolution} = Config.meetings;
-  const shareConstraints =
-    options.sharePreferences && options.sharePreferences.shareConstraints ?
-      options.sharePreferences.shareConstraints :
-      {
-        cursor: 'always',
-        aspectRatio: Config.meetings.aspectRatio,
-        ...(options.sharePreferences && options.sharePreferences.highFrameRate ?
-          {
-            frameRate: Config.meetings.videoShareFrameRate,
-            height: Config.meetings.resolution.idealHeight,
-            width: Config.meetings.resolution.idealWidth
-          } :
-          {
-            frameRate: Config.meetings.screenFrameRate,
-            height: resolutionConf.idealHeight || screenResolution.idealHeight,
-            width: resolutionConf.idealWidth || screenResolution.idealWidth
-          }),
-        // Leave last to apply overrides
-        ...resolutionConf
-      };
+  // SDK screen share resolution settings from Webex.init
+  const customResolution = config.screenResolution || {};
+  // user defined share preferences
+  const hasSharePreferences = options.sharePreferences;
+  // eslint-disable-next-line max-len
+  const hasCustomConstraints = hasSharePreferences && hasSharePreferences.shareConstraints;
+  // eslint-disable-next-line max-len
+  const hasHighFrameRate = hasSharePreferences && hasSharePreferences.highFrameRate;
+  const {
+    screenResolution,
+    resolution,
+    videoShareFrameRate,
+    screenFrameRate,
+    aspectRatio
+  } = Config.meetings;
+
+  let shareConstraints = {
+    cursor: 'always',
```

Comment:
constant>

Reply:
Agreed with @chrisadubois, let's convert this to a constant.

Reply:
Good catch, I missed this when I refactored. I not a fan of magic numbers or strings.